### ISSUE TITLE
Service doesn't appear to be working anymore

### ISSUE THREAD

**Issue by jayphelps** [mbebenita/WasmExplorer#11]
I use this tool quite a bit, noticed this morning that it now always outputs the same, basically empty wast and the Firefox assembly section is always completely empty.

```
(module
  (table 0 anyfunc)
  (memory $0 1)
  (export "memory" (memory $0))
)
```

https://goo.gl/L6lPs0

Tried on two computers, so I don't believe it's just me. No errors in the console or HTTP requests and it does still compile the code cause malformed programs produce compiler errors.

**Comment by yurydelendik** [comment]
Fixed with service v2.2

**Comment by jayphelps** [comment]
@yurydelendik thanks for quick turnaround. 🎉
